src/ser-datasets.json

```diff
@@ -1,4 +1,74 @@
 {
```

Comment:
This looks great but please make sure that the Datasets added in JSON still follow the right chronological order (by year).

Reply:
@SuperKogito I thought that chronological order is by input. Since we can "sort by year"  in [the rendered page](https://superkogito.github.io/SER-datasets/), there is no point to manually sort by year, isn't it?